jetty-core/jetty-start/src/main/java/org/eclipse/jetty/start/StartArgs.java

```diff
@@ -247,25 +252,25 @@ private void addFile(Module module, String uriLocation)
             return;
         }
 
-        Environment environment = getEnvironment(module);
+        StartEnvironment environment = getEnvironment(module);
         FileArg arg = new FileArg(module, environment.getProperties().expand(uriLocation));
         if (!files.contains(arg))
             files.add(arg);
     }
 
-    private Environment getEnvironment(Module module)
+    private StartEnvironment getEnvironment(Module module)
     {
         String envName = module == null ? null : module.getEnvironment();
-        Environment environment = envName == null ? getCoreEnvironment() : getEnvironment(envName);
+        StartEnvironment environment = envName == null ? getJettyEnvironment() : getEnvironment(envName);
         return environment;
     }
 
     public void dumpJavaEnvironment(PrintStream out)
     {
         // Java Details
         out.println();
-        out.println("Java Environment:");
-        out.println("-----------------");
+        out.println("Key System Properties:");
```

Comment:
What does "Key" mean here? Perhaps "JVM version & properties" so that below we can do "Jetty version & properties"?